examples/objects/ui.py

```diff
@@ -0,0 +1,77 @@
+from arena import *
+
+scene = Scene(host="arenaxr.org", scene="example")
+
+prompt = None
+
+
+@scene.run_once
+def setup_scene():
+    # Add a simple info card with text and image
+    intro_card = Card(
+        object_id="intro-card",
+        persist=True,
+        title="Welcome to ARENA",
+        body="ARENA is a framework designed to both simplify and host collaborative "
+        "multi-user XR applications. It makes it easy to combine virtual and "
+        "physical components like sensors, actuators, and digital interfaces in "
+        "an immersive 3D environment.",
+        bodyAlign="left",
+        imgDirection="left",
+        img="/static/landing/images/xr-logo-v8.png",
+        imgSize="contain",
+        position=Position(0, 2, -2.5),
+    )
+    scene.add_object(intro_card)
+
+    # Add a popup prompt with single button
+
+    def prompt_handler(_scene, evt, _msg):
+        if evt.type == "buttonClick":
+            if evt.data.buttonName == "OK":
+                print("OK clicked!")
+                scene.delete_object(prompt)
+
+    # Add a button panel, with two sets of buttons
+
+    # Legacy buttons list supports strings only
```

Comment:
Note that current prod does not have release that handles dict/json button elements (required for image buttons)

Until that is released, use the legacy list of strings below

Reply:
Prod is now running image-button supported scehma